client/reader.js

```diff
@@ -31,6 +31,9 @@ var apikey = nconf.get('apikey');
 var identifier = nconf.get('identifier');
 var sendFunctionCode = nconf.get('sendFunctionCode') || false;
 var useTimestamp = nconf.get('useTimestamp') || true;
+var EAS_Exclude_Events = nconf.get('EAS_Exclude_Events');
+var EAS_Include_FIPS = nconf.get('EAS_Include_FIPS');
+var EAS_sendType = nconf.get('useTimestamp') || true;
```

Comment:
Wrong variable :)

Reply:
Fixed with [2ccefa7ec0c4f39ccbf37175f0657b9bce74ed01](https://github.com/pagermon/pagermon/pull/435/commits/2ccefa7ec0c4f39ccbf37175f0657b9bce74ed01)

``        var decodedMeassage = SAME.decode(line, EASOpts.excludeEvents, EASOpts.includeFIPS);          // Returns a object with all the info
```